src/core/NewDomain.ml

```diff
@@ -1275,7 +1276,12 @@ and Face :
       (r, r', Delayed.make @@ f (r, r', Delayed.drop_rel bdy))
   end
 
-and Abs : functor (X : Domain) -> Domain with type t = X.t abs =
+and Abs : functor (X : Domain) ->
+  sig
+    include Domain with type t = X.t abs
+    val inst : rel -> t -> dim -> X.t
```

Comment:
@jonsterling New, simple API to remove misconception of `run` inside an abstraction.

Reply:
:+1: